R/antigen.garnish_predict.R

```diff
@@ -302,7 +335,7 @@ parallel::mclapply(dt[, spc %>% unique], function(s){
                                                  "iedb_uuid",
                                                  "IEDB_anno",
                                                  "iedb_score")],
-           by = c("nmer_uuid"),
+           by = c("nmer_uuid", shortmerge),
```

Comment:
did you text when shortmerge loop is skipped? (== null)?

Reply:
as in did I message the console?

Reply:
sorry, typo.

does this statement work when short merge is never changed from default value of `NULL`? not clear to me that it does

Reply:
I tested and didn't have any problems when shortmerge was NULL or had a column name as value.